src/header.tsx

```diff
@@ -162,12 +164,16 @@ const ViewSelector = ({ isGrid, setIsGrid, sortBy, setSortBy, showHidden, setSho
     }) => {
     const [isOpen, setIsOpen] = useState<boolean>(false);
     const onToggleClick = (isOpen: boolean) => setIsOpen(!isOpen);
+    const dialogs = useDialogs();
+
     const onSelect = (_ev?: React.MouseEvent, itemId?: string | number) => {
         if (itemId === "hidden-toggle") {
             setShowHidden(prevShowHidden => {
                 localStorage.setItem("files:showHiddenFiles", !showHidden ? "true" : "false");
                 return !prevShowHidden;
             });
+        } else if (itemId === "shortcuts-help") {
```

Comment:
This added line is not executed by any test.

Reply:
dw it is tested :)